cioos_data_transform/cioos_data_transform/utils/odf.py

```diff
@@ -14,6 +15,17 @@
 """
 
 
+def odf_dtypes(input=None):
```

Comment:
`input` should be renamed- how would you describe the purpose/use of the variable to someone?

Reply:
how about odf_datatype_mapping_python